action.yml

```diff
@@ -59,13 +59,32 @@ runs:
       uses: ublue-os/remove-unwanted-software@v6
       if: ${{ inputs.maximize_build_space == 'true' }}
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+      with:
+        install: true
+        cache-binary: ${{ inputs.use_cache }}
+
+    - name: Install BlueBuild
+      shell: bash
+      env:
+        CLI_VERSION_TAG: ${{ inputs.use_unstable_cli == 'true' && 'main' || 'v0.8.2' }}
+      run: |
+        docker run \
+          --detach \
+          --rm \
+          --name blue-build-installer \
+          ghcr.io/blue-build/cli:${{ env.CLI_VERSION_TAG }}-installer \
+          tail -f /dev/null
+        docker cp blue-build-installer:/out/bluebuild /usr/local/bin/bluebuild
+        docker stop -t 0 blue-build-installer
+
     # clones user's repo
     - uses: actions/checkout@v4
 
-    # deps used by cli to build & inspect imaegs
-    - name: Install Dependencies
-      shell: bash
-      run: sudo apt-get install -y podman
+    - name: Expose GitHub Runtime
```

Comment:
What's the purpose of this? Should be documented as a comment at least.

Reply:
This is required in order to take advantage of docker's cache integration with GitHub Actions.

https://github.com/moby/buildkit?tab=readme-ov-file#github-actions-cache-experimental

Reply:
should be resolved in: https://github.com/blue-build/github-action/pull/29/commits/bf474dcf5124b6cfbab44f9b68b87fc4b5b0702a